client/layout/oauth-client/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const OauthClientLayout = ( { client } ) => (
+	<div className="oauth-client">
+		<nav>
```

Comment:
Is it necessary to have this be nested inside a `<div>`? If we're trying to use this tag semantically, it might make more sense to put it as the topmost element.